airflow/decorators/__init__.pyi

```diff
@@ -41,7 +41,6 @@ class TaskDecoratorFactory:
         **kwargs,
     ) -> TaskDecorator:
         """Create a decorator to convert the decorated callable to a task.
-
```

Comment:
Please don’t remove the blank line (and please add one for `branch`). Those are intended to separate the “summary” and other descriptions.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Restored blank lines.